packages/design-system/src/components/N8nCheckbox/Checkbox.vue

```diff
@@ -63,4 +64,13 @@ export default Vue.extend({
 </script>
 
 <style lang="scss" module>
+.n8nCheckbox {
```

Comment:
This is added to force checkbox label text to wrap and align properly even if it's longer then one line. The only other place it's used is in [FormInputs component storybook](http://localhost:6006/?path=/story/modules-forminputs--form-inputs) (it can be tested with diferent param values there).